specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -363,7 +363,7 @@
           }
         },
         "x-ms-pageable": {
-          "nextLinkName": "nextLink"
+          "nextLinkName": null
```

Comment:
Why setting it to `null`? The response does have a `nextLink` property.

Reply:
Can you please help with this @ha0ch1

Reply:
@gary-x-li Thanks for brining this up, I've removed the next link property in the response since this API is not expected to return a nextlink property. 

Here is the PR for the fix, could you please take a look? https://github.com/Azure/azure-rest-api-specs/pull/28696/commits/87a53953594d401f60fd762c2c1bb5779b12ce5d

Reply:
This will be resolved with a backend change, not a swagger change. This has been reverted.